scripts/src/models/all.ts

```diff
@@ -1,83 +1,111 @@
-});
+	created_date: Sequelize.DATE,
+	expire_date: Sequelize.DATE, // set to 2 weeks? session period?
+	valid: Sequelize.BOOLEAN, // I can invalidate this token manually?
+	token: Sequelize.STRING,
+}));
+
 
-const Application = sequelize.define("applications", {
-	id: Sequelize.STRING,
+const Application = models.push(sequelize.define("applications", {
+	id: { type: Sequelize.STRING, primaryKey: true },
 	name: Sequelize.STRING,
 	jwt_public_key: Sequelize.STRING,
-	wallet_address: Sequelize.STRING, // kre goes here
 	created_date: Sequelize.DATE, 
-});
+}));
 
-const OfferOwner = sequelize.define("offer_owners", {
-	id: Sequelize.STRING,
+const OfferOwner = models.push(sequelize.define("offer_owners", {
+	id: { type: Sequelize.STRING, primaryKey: true },
 	name: Sequelize.STRING,
-})
+}));
 
-const Offer = sequelize.define("offers", {
-	id: Sequelize.STRING,
+const Offer = models.push(sequelize.define("offers", {
+	id: { type: Sequelize.STRING, primaryKey: true },
 	owner_id: Sequelize.STRING, // OfferOwner
 	meta: Sequelize.JSON, // title, description, image
 	type: Sequelize.ENUM("spend", "earn"),
-	amount: Sequelize.INTEGER,  // amount in kin? min-kin, micro-kin? teds?
-	content: Sequelize.JSON, // in case of poll, the poll questions and answers. in case of coupon - empty?
+	amount: Sequelize.BIGINT,  // amount in micro-kin
 	cap: Sequelize.JSON, // complex object with cap rules
 	created_date: Sequelize.DATE,
-});
+}));
 
-const AppOffer = sequelize.define("app_offers", {
-	app_id: Sequelize.STRING,
+const OfferContent = models.push(sequelize.define("offer_content", {
 	offer_id: Sequelize.STRING,
-});
+	content: Sequelize.JSON
+}));
 
-const ApiToken = sequelize.define("api_tokens", {
-	user_id: Sequelize.STRING,
-	app_id: Sequelize.STRING,  // redundent
-	created_date: Sequelize.DATE,
-	expire_date: Sequelize.DATE, // set to 2 weeks? session period?
-	valid: Sequelize.BOOLEAN, // I can invalidate this token manually?
-	did_tos: Sequelize.BOOLEAN,
-	token: Sequelize.STRING,
-});
+const AppOffer = models.push(sequelize.define("app_offers", {
+	app_id: Sequelize.STRING,
+	offer_id: Sequelize.STRING,
+}));
 
 // pre-bought coupons
-const Asset = sequelize.define("assets", {
-	id: Sequelize.STRING,
+const Asset = models.push(sequelize.define("assets", {
+	id: { type: Sequelize.STRING, primaryKey: true },
 	type: Sequelize.STRING,
 	value: Sequelize.JSON,
 	is_used: Sequelize.BOOLEAN,
 	created_date: Sequelize.DATE,
-});
+}));
 
-const TransactionHistoryItem = sequelize.define("transaction_history_item", {
+const Transaction = models.push(sequelize.define("transactions", {
 	order_id: Sequelize.STRING,
 	blockchain_txid: Sequelize.STRING,
 	user_id: Sequelize.STRING, // redundent
 	meta: Sequelize.JSON, // some human readable data for UI
 	type: Sequelize.STRING,
 	value: Sequelize.JSON,
 	created_date: Sequelize.DATE,
-});
+}));
+
+models.sync().then(synced => console.log(`finished syncing all models (${ synced.length })`));
```

Comment:
I'd prefer everything related to sync to be in a separate file even if that creates the overhead of having to remember to add the models to the list of synced classes

Reply:
it's a start. we'll obviously change things.  It was just a simple way to make sure that no model is left un-synced.

Reply:
@liorama said to put int npm scripts